lab-bundle-spec.md

```diff
@@ -50,6 +50,7 @@ description:
     en: "No, seriously. It's the best lab ever. You're going to love it!"
 
 duration: 60
+max_duration: 60
```

Comment:
i know it's just an example, but would be good to use different values so that creators don't think they're supposed to be the same thing
```suggestion
max_duration: 90
```